Source: Upstream SVN, r860334
Upstream: Yes
Reason: Binaries weren't installed with CMake 2.6

Index: branches/work/lancelot/kde4.1-backport/CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 860333)
+++ CMakeLists.txt	(revision 860334)
@@ -1,4 +1,5 @@
-cmake_minimum_required(VERSION 2.6)
+# cmake_minimum_required(VERSION 2.6)
+find_package(KDE4 REQUIRED)
 
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 
